pkg/kubelet/kubelet.go

```diff
@@ -901,6 +901,23 @@ func (kl *Kubelet) initialNodeStatus() (*api.Node, error) {
 		if err != nil {
 			return nil, err
 		}
+
+		// If the cloud has zone information, label the node with the zone information
+		zones, ok := kl.cloud.Zones()
```

Comment:
Cloud providers may have placement abstractions that do not clearly map to other cloud providers, in which case labels may make more sense.  For instance, AWS has the concept of placement groups in addition to regions and zones.